hw/mcu/dialog/da1469x/src/da1469x_clock.c

```diff
@@ -286,6 +336,17 @@ da1469x_clock_lp_rc32k_calibrate(void)
     g_mcu_clock_rc32k_freq = freq;
 }
 
+void
+da1469x_clock_lp_xtal32k_calibrate(void)
+{
+#if MYNEWT_VAL(MCU_CLOCK_XTAL32K_ALLOW_CALIB)
+    g_mcu_clock_xtal32k_freq =
+        da1469x_clock_calibrate(2, MYNEWT_VAL(MCU_CLOCK_RCX_CAL_REF_CNT));
+#else
+    g_mcu_clock_xtal32k_freq = XTAL32M_FREQ;
```

Comment:
XTAL32K_FREQ

Reply:
One of these typos still managed to creep through - good find!

Reply:
Fixed by amending the original commit.